### DISCUSSION TITLE
Meeting minutes for 18.07.2024

### DISCUSSION THREAD

**Discussion by konradweiss** [Fraunhofer-AISEC/cpg#1616] [Dev Meetings]
### Agenda

1. Open PRs
2. Next CPG release
3. Can we get rid `scopeManager.addDeclaration` with having `withChildren` now?

### Discussed PRs

- [x] #1613 
- [ ] #1608
- [ ] #1479 
- [ ] #1402 
- [ ] #1606
...

**Comment by oxisto** [comment]
#1402 was already discussed to be "finished". It still needs a review or at least an approval :)

**Comment by oxisto** [comment]
https://github.com/Fraunhofer-AISEC/cpg/milestones?with_issues=no

**Comment by konradweiss** [comment]
### Agenda

2. Next CPG release
- AST-Parent Edges
- We can create a milestone for 9.0 and future large releases to collect issues.
- Python imports: done
- Type Name normalization: done
- That long Pr Name with resolution: Last adaptations can be made in the future. #1608 
  This PR has review prio
- #1609 cosmetic changes
- For now we do not get rid of `MemberCallExpression`

4. Can we get rid `scopeManager.addDeclaration` with having `withChildren` now?
5. Can we optimize the replacement of nodes and entire subgraphs?


### Discussed PRs

- [x] #1613 
- [x] #1608
- [x] #1479 
- [x] #1402 
- [x] #1606
...
